src/main/java/io/mvnpm/esbuild/resolve/BaseResolver.java

```diff
@@ -19,14 +19,23 @@
 import org.apache.commons.compress.compressors.gzip.GzipCompressorInputStream;
 
 public abstract class BaseResolver {
-    public static final String EXECUTABLE_PATH = "package/bin/esbuild";
+    private static final String EXECUTABLE_PATH = "package/bin/esbuild";
+    private static final String WINDOWS_EXE_PATH = "package/esbuild.exe";
 
     protected Resolver resolver;
 
     public BaseResolver(Resolver resolver) {
         this.resolver = requireNonNull(resolver, "resolver is required");
     }
 
+    public static String executablePath() {
```

Comment:
@edewit you could have kept it static (to avoid resolving it many times), the os doesn't change.